packages/ui-library/src/theme/typography.ts

```diff
@@ -179,20 +69,15 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     lineHeight: 1.35, // 21.6/16
     color: colors[mode].typography.typo2,
   },
-  body1: {
+  bodyBold: {
```

Comment:
Seems like body, bodyBold, bodySemiBold all inherit from the same `body` which defines the same lineHeight fontStyle, sfontSize and fontFamily as all the rest, we could create that base one, and make all the rest of this extend that one